vsm

```diff
@@ -22,6 +23,34 @@ LOGIC_REPLACE = {'||': 'or',
 
 KEYWORDS = ['condition', 'emit']
 
+start_time_ms = 0
+
+class Log(object):
+    '''
+        Utility class for logging messages
+    '''
+
+    '''
+        Log an informative (non-error) message
+    '''
+    @staticmethod
+    def i(msg, timestamp=False):
```

Comment:
I have mixed feelings about the one character names, but seeing them in use I lean more towards liking them =)